File: app/src/main/java/org/xtimms/yomu/interfaces/FilterCallback.java
package org.xtimms.yomu.interfaces;

import org.xtimms.yomu.models.MangaGenre;
import org.xtimms.yomu.models.MangaType;

public interface FilterCallback {

    void setFilter(int sort, int additionalSort, MangaGenre[] genres, MangaType[] types);
}

File: app/src/main/java/org/xtimms/yomu/interfaces/FilterCallback.kt
package org.xtimms.yomu.interfaces

import org.xtimms.yomu.models.MangaGenre
import org.xtimms.yomu.models.MangaType

interface FilterCallback {

    fun setFilter(
        sort: Int,
        additionalSort: Int,
        genres: Array<MangaGenre?>?,
        types: Array<MangaType?>?
    )

}